services/QuillLMS/client/app/bundles/Shared/hooks/__tests__/useScaledTableColumnWidths.test.tsx

```diff
@@ -103,4 +103,51 @@ describe('useScaledTableColumnWidths', () => {
 
     window.innerWidth = original; // reset
   });
+
+  it('respects additionalWidthRestrictions when calculating container width', () => {
+    const ref = mockContainer(1200);
+    const { result } = renderHook(() =>
+      useScaledTableColumnWidths({
+        additionalWidthRestrictions: 100,
+        columnWidths: [
+          { width: 300 },
+          { width: 300 },
+          { width: 300 },
+          { width: 300 },
+        ],
+        containerRef: ref,
+        maxTableWidth: 1200,
+        minTableWidth: 963,
+      })
```

Comment:
The difference between this big object and the one in the test below is pretty subtle.  I wonder if there's a way to define the base object at a higher level, and then add the `additionalWidthRestrictions` key by itself inside this case so it's obvious what the difference between the two cases is.